examples/lines_bars_and_markers/scatter_hist.py

```diff
@@ -30,33 +70,61 @@
 rect_histx = [left, bottom_h, width, 0.2]
 rect_histy = [left_h, bottom, 0.2, height]
 
-# start with a rectangular Figure
-plt.figure(figsize=(8, 8))
+# start with a rectangular Figure and add axes to it
```

Comment:
All figures are rectangular :smile:. You probably mean "square". But IMHO this is not an important property here. It would work for any figure. This is just an implementation detail to make the plot more nice and does not need commenting.

Similar "add axes to it" is redundant. The code already says it `fig.add_axes`.

Reply:
Yes, square is of course what is meant here. The figure being square is important, else, it wouldn't look as nice as it does. For non-square figures one would need to play with different spacings in `rect_hist{x,y}` (or for the other example with different `wspace` / `hspace` of the gridspec).